src/net.cpp

```diff
@@ -487,6 +487,26 @@ void CConnman::AddWhitelistPermissionFlags(NetPermissionFlags& flags, const CNet
     }
 }
 
+std::string CNode::ConnectionTypeAsString() const
```

Comment:
μNit, feel free to ignore: the `As` feels redundant. (I.e. `ConnectionTypeString()`)

Reply:
Nah, I propsed the current naming. Lets not bikeshed it too much.